electron/locale/strings.js

```diff
@@ -49,6 +49,7 @@ string.menuQuit = 'Quit Wire';
 string.menuShowHide = 'Show/Hide Menu';
 string.menuSavePictureAs = 'Save Picture As...';
 string.menuNoSuggestions = 'No Suggestions';
+string.menuAutoStart = 'Start with OS';
```

Comment:
Sure thing! Should I make the change in just `strings.js` or should I also change `strings-en.js`? I think the `string-en.js` is provided by Crowdin, right?